modules/acquia_cms_common/tests/src/ExistingSite/ContentTypeListTestBase.php

```diff
@@ -382,7 +382,7 @@ protected function getQuery() : QueryInterface {
   /**
    * {@inheritdoc}
    */
-  public function tearDown() {
+  public function tearDown() : void {
```

Comment:
@deepakmishra2 Covered in this please remove from this change ~~ https://github.com/acquia/acquia_cms/pull/1356/files#diff-1b1f03796fd04b1a3ea69c3a7b0054be5134cda91f94bdb4d7661ffa7408a4dbR385

cc: @rajeshreeputra